### PR TITLE
system_monitor_ros in ROS2: initial commit and working setup

### PR DESCRIPTION
**Pull request by TSC21**
This PR just serves to show the changes from one branch to the other so the Avoidance team is aware of the porting process.

The branch is fully functional and tested locally with PX4 SITL. Currently just missing an integration test to be run on Travis CI (as the build is already in CI).

### CODE REVIEW

**Comment by TSC21** [comment]
CI is failing because we need `ros2bag` on the containers. Added: https://github.com/PX4/containers/pull/246.

**Comment by TSC21** [comment]
@caleridas although I agree with with your review, most of the code you are reading/commenting was written by @Jaeyoung-Lim and @baumanta, which I didn't want to change. The sole purpose of this was to update the ROS1 code of the system monitor to use the ROS2 API. The sole functionality of it wasn't changed at all. Could that be addressed in further PR's? And I ask this because your change requests are focused on the existing API which you can find on the master branch (ROS1 API used though).

File: tools/fix_style.sh [hunk 1]
```diff
@@ -13,9 +13,9 @@ fi
 for arg in "$@"
 do
     if [ -f $arg ]; then
-        clang-format-3.8 -i -style='{BasedOnStyle: google, ColumnLimit: 120}' $arg
+        clang-format-6.0 -i -style='{BasedOnStyle: google, ColumnLimit: 120}' $arg
     elif [ -d $arg ]; then
-        find $arg -iname '*.h' -o -iname '*.cpp' -o -iname '*.hpp' | xargs clang-format-3.8 -i -style='{BasedOnStyle: google, ColumnLimit: 120}'
+        find $arg -iname '*.h' -o -iname '*.cpp' -o -iname '*.hpp' | xargs clang-format-6.0 -i -style='{BasedOnStyle: google, ColumnLimit: 120}'
```
Comment (Jaeyoung-Lim) [inline tools/fix_style.sh]: Not related to this PR, but do we want to also change PX4/avoidance to clang-format-6.0?

**Comment by TSC21** [comment]
> @TSC21 Have you checked if this also gets logged properly in the uLogs?
Yes:
Screenshot from 2020-01-17 11-48-13

File: tools/fix_style.sh (hunk 1 above)
Comment (TSC21) [reply tools/fix_style.sh]: Eventually yes, since only 3.9 and above are available at Ubuntu 18.04.

**Comment by TSC21** [comment]
As per @LorenzMeier suggestion, https://github.com/Auterion/system_monitor_ros/tree/ros1 is now a direct clone of master and this PR can be merged, as master becomes ROS 2 compatible.

**Comment by Jaeyoung-Lim** [comment]
@TSC21 Thanks!

**Pull request merged**